inc/field.class.php

```diff
@@ -81,7 +81,7 @@ public function prepareQuestionInputForTarget($input) {
    public function show($canEdit = true) {
       $required = ($canEdit && $this->fields['required']) ? ' required' : '';
 
-      echo '<div class="form-group ' . $required . '" id="form-group-field' . $this->fields['id'] . '">';
+      echo '<div class="form-group ' . $required . '" id="form-group-formcreator_field_' . $this->fields['id'] . '">';
```

Comment:
there is a mismatch between the ID of the tag and the key of the values returned by ajax/showdields.php